installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,220 @@
+====
+* You downloaded the AWS CLI and installed it on your computer. See link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or UNIX)] in the AWS documentation.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[manually create and maintain IAM credentials].
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+[id="installation-requirements-user-infra_{context}"]
+== Requirements for a cluster with installer-provisioned infrastructure
+
+For a cluster that contains installer-provisioned infrastructure, the Machine API will be responsible for creating the worker nodes declared by the MachineSets available on `<install-dir>/openshift/99_openshift-cluster-api_worker-machineset-*.yaml`
+
+This section describes the requirements for deploying {product-title} on an existing VPC with Local Zone subnets.
+
+include::modules/installation-machine-requirements.adoc[leveloffset=+2]
+include::modules/installation-minimum-resource-requirements.adoc[leveloffset=+2]
+include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
+include::modules/installation-aws-arm-tested-machine-types.adoc[leveloffset=+2]
+
+include::modules/installation-aws-marketplace-subscribe.adoc[leveloffset=+1]
+
+//These next four modules seem like they're placed awkwardly, but you need the subnets to put in the install-config.yaml file.
+include::modules/installation-creating-aws-vpc-localzone.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-vpc-localzone.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+include::modules/installation-creating-aws-subnet-localzone.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-subnet-localzone.adoc[leveloffset=+2]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
+
+include::modules/installation-disk-partitioning-upi-templates.adoc[leveloffset=+2]
```

Comment:
I think this optional also can be removed.
```suggestion
```

Reply:
Deleted, thank you!